### ISSUE TITLE
list of supported repository types?

### ISSUE THREAD

**Issue by gliptak** [eclipse/packagedrone#75]
I might have missed this in the Wiki

https://wiki.eclipse.org/PackageDrone

but I didn't find a list of supported repository types (Maven, Gradle, RPM, DEB, etc.)

They are listed on:

https://binary-repositories-comparison.github.io/

**Comment by ctron** [comment]
True. There still is the list of features, which should list those [1]

[1] https://packagedrone.org/features/
